src/app/public/public_api.ts

```diff
@@ -1,3 +1,4 @@
+export * from './modules/dropdown/types/dropdown-horizontal-alignment';
```

Comment:
☝️ Not part of this issue, but this property is in our docs. Probably should have been exported before.